internal/server/placepage/place_page.go

```diff
@@ -244,11 +247,42 @@ func fetchBtData(
 		return nil, nil, nil, err
 	}
 
+	// Fetch categories data
+	btCategoryData, err := bigtable.Read(
+		ctx,
+		store.BtGroup,
+		bigtable.BtLandingPageCategories,
+		[][]string{places},
+		func(jsonRaw []byte) (interface{}, error) {
+			var categories pb.Categories
+			if err := proto.Unmarshal(jsonRaw, &categories); err != nil {
+				return nil, err
+			}
+			return &categories, nil
+		},
+	)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	categoryData := map[string]*pb.Categories{}
+	for _, place := range places {
+		categoryData[place] = &pb.Categories{Category: []string{}}
```

Comment:
I don't know where else Categories is used, but it seems it is internal to mixer?

The proto corresponding to this is http://google3/knowledge/answers/stats/proto/chart_store.proto;l=80-84;rcl=485168034

Also, should we mark this reserverd: https://github.com/datacommonsorg/mixer/blob/34989caa16f7ba773fd2cfc090e78e5a155f5801/proto/internal.proto#L83 ?

Reply:
I updated the proto to match the internal one.